app/recurring_expenses/views.py

```diff
@@ -123,6 +123,52 @@ def create_recurring_expense(request):
     return render(request, "recurring_expenses/new.html", {"form": form})
 
 
+@login_required
+def export_recurring_expenses(request):
+    if "csv" in request.GET["format"]:
```

Comment:
A couple of small things

I think the default behavior is currently undefined.

What happens if no format is requested at all?
What happens if the requested format is given in uppercase or mixed case?
What happens if someone manually requests a format which is not supported? e.g. xml or "foobar"

I think the first two are fairly easy fixes

1. We set a default format
    ```python
    format = request.GET.get("format", "csv")
    ```
 
2.  Convert format to lowercase for comparison
     ```python
     if format.lower() == 'csv'
     ```

    or as you have written it, 
    
    ```python
    if 'csv' in format.lower()
    ```

Reply:
The third case we will just have to throw an error anyways

Reply:
Third case would be covered by an empty response currently
```
else:
        response = HttpResponse(status=204)
```

Reply:
So for:
```
format = request.GET.get("format", "csv")
```
`csv` is set for format if no format is provided?